api-docs/cloud/ref.yml

```diff
@@ -5031,23 +5199,83 @@ components:
       type: object
     Template:
       items:
+        description: |
+          A template entry.
+          Defines an InfluxDB resource in a template.
         properties:
           apiVersion:
+            example: influxdata.com/v2alpha1
             type: string
           kind:
             $ref: '#/components/schemas/TemplateKind'
-          meta:
+          metadata:
+            description: >
+              Metadata properties used for the resource when the template is
+              applied.
             properties:
               name:
                 type: string
             type: object
           spec:
+            description: >
+              Configuration properties used for the resource when the template
+              is applied.
+
+              Key-value pairs map to the specification for the resource.
+
+
```

Comment:
This is an artifact of the processor. Our standard is to use `description: |` for multiline strings in the spec. That allows us to use single empty lines when that's what we want in the processed result (WYSIWYGish). However, our processor translates this to `description: >` and double-empty lines. In short, it's probably best to ignore the spacing in the `.yml` files and just review the HTML formatted output in the site (e.g. on localhost or staging).

Reply:
> Notice after description sometimes we have 2 spaces, and other times one or none, in case we had a standard

This is an artifact of the processor. Our standard is to use description: | for multiline strings in the spec. That allows us to use single empty lines when that's what we want in the processed result (WYSIWYGish). However, our processor translates this to description: > and double-empty lines. In short, it's probably best to ignore the spacing in the .yml files and just review the HTML formatted output in the site (e.g. on localhost or staging).